### PR TITLE
Community details page update 

### PR DESCRIPTION
**Pull request by LHinson**
Updated the details for the next community meeting detail as well as added links forSeptember's meeting assets.

### CODE REVIEW

**Comment by LHinson** [comment]
@dlabrecq @bleathem @andresgalante can I get a couple reviews here? 

bitmoji

**Comment by dlabrecq** [comment]
LGTM.

**Comment by bleathem** [comment]
:+1:

File: source/community/monthly-community-meeting/index.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ layout: page
   </thead>
   <tbody>
   <tr>
+    <td>September 7, 2016</td>
+    <td><a href="https://www.dropbox.com/s/fp3p58n9wmfgnvt/PatternFly%20Community%20Meeting%202016-09.pdf?dl=0">View Deck</a></td>
+    <td><a target="_blank" href="https://bluejeans.com/s/aNH4r/">View Recording</a></td>
+  </tr>
+  <tr>
+  <tr>
```
Comment (cardosogabriel) [inline source/community/monthly-community-meeting/index.md]: Isn't this one an unnecessary <tr>?

Comment (dlabrecq) [inline source/community/monthly-community-meeting/index.md]: Good catch. There appears to be two extra <tr> tags if you view the full source.

**Pull request merged**